app/locales/ar.json

```diff
@@ -66,59 +63,22 @@
     "launch_screen4_subheader": "إذا كان الاختبار إيجابيًا ، انت فقط يمكنك اختيار المشاركة",
     "launch_set_up_phone": "إعداد هاتفي",
     "legal_page_title": "النظام القانوني",
-    "less_than_one_minute": "أقل من دقيقة واحدة",
     "loading_public_data": "جاري تحميل البيانات...",
     "location_disabled_message": "يتطلب تطبيق COVID Safe Paths خدمات الموقع",
     "location_disabled_title": "تم تعطيل خدمة تتبع الموقع",
     "location_enabled_message": "يقوم تطبيق COVID Safe Paths بتخزين إحداثيات موقعك بأمان مرة واحدة كل خمس دقائق على هذا الجهاز",
     "location_enabled_title": "تم تفعيل تطبيق COVID Safe Paths",
-    "maps_import_button_text": "تحميل المواقع المتعقبة السابقة",
-    "maps_import_disclaimer": "لا يرتبط تطبيق Safe Paths بـ Google ولا يشارك بياناتك مطلقًا",
-    "maps_import_text": "لمعرفة ما إذا كنت قد تعرفت على شخص لديه COVID-19 قبل تحميل هذا التطبيق ، يمكنك استيراد سجل موقعك الشخصي",
-    "maps_import_title": "خرائط جوجل",
-    "nCoV2019_url_info": "لمزيد من المعلومات حول مجموعة البيانات لهذه الخريطة",
+    "logging_active": " خدمة تتبع الموقع مفعلة",
+    "logging_inactive": "خدمة تتبع الموقع غير مفعلة",
     "news_subtitle": "اقرأ عن آخر تحديثات COVID من السلطات الصحية الخاصة بك وبشكل عام",
```

Comment:
I just noticed this.. we have COVID-19 everywhere else. This is the only key that doesn't have this. we should fix this.

Reply:
You can add a comment in lokalise